package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ng-iotile-cloud",
-  "version": "0.12.9",
+  "version": "0.13.0",
```

Comment:
Not sure if you have a reason to use 0.13.0 instead of 0.12.10, which is more consistent with previous changes.

But the fact is that we have been using versions wrong (my mistake), so good time to start doing this right.

Going forward (and until we declare it 1.0.0), and assuming the same Major.Minor.Patch numbering, we should use:

- Minor: Every time there is new functionality. New API, remove API, change of behavior in API
- Patch: Every time we are just fixing a bug. Note this can still change functionality, but if the old functionality was basically always wrong, then we can call it a patch.

Reply:
Sorry, I was assuming that 0.12.10 is equivalent to 0.12.1.